web-registry/src/components/organisms/BuyCreditsModal/BuyCreditsModal.utils.ts

```diff
@@ -39,15 +52,35 @@ export const getOptions = ({
 }: GetOptionsParams): Option[] => {
   if (!project?.sellOrders?.length) return [];
 
-  const sellOrderOptions = project?.sellOrders.map(sellOrder => {
-    return {
-      label: getSellOrderLabel({ sellOrder, allowedDenomsData }),
-      value: sellOrder.id,
-    };
+  const retirableSellOrders = project.sellOrders.filter(
+    sellOrder => !sellOrder.disableAutoRetire,
+  );
+  const retirableAndTradableSellOrders = project.sellOrders.filter(
+    sellOrder => sellOrder.disableAutoRetire,
+  );
+
+  const retirableOptions = getSellOrdersOptions({
+    sellOrders: retirableSellOrders,
+    allowedDenomsData,
   });
-  return sellOrderOptions.length > 1
-    ? [{ label: 'Choose a sell order', value: '' }, ...sellOrderOptions]
-    : sellOrderOptions;
+
+  const retirableAndTradableOptions = getSellOrdersOptions({
+    sellOrders: retirableAndTradableSellOrders,
+    allowedDenomsData,
+  });
+
+  const allOptionsLength =
+    retirableOptions.length + retirableAndTradableOptions.length;
+
+  return allOptionsLength > 1
+    ? [
+        { label: 'Choose a sell order', value: '', disabled: true },
+        { label: 'RETIRABLE ONLY', value: '', disabled: true },
+        ...retirableOptions,
+        { label: 'TRADABLE AND RETIRABLE', value: '', disabled: true },
```

Comment:
@erikalogie I know I told you I liked "TRADABLE" better, but now I'm feeling like "TRADEABLE" and "RETIREABLE" might be better? IDK, probably we should ask more people for feedback.

Reply:
@mhagel we use "tradable" everywhere in the ledger docs so I think it would be good for consistency: https://buf.build/regen/regen-ledger/docs/main:regen.ecocredit.basket.v1

Reply:
learn something new every day, i would have voted for the versions with the E's but they do mean the same thing.